_data/wai-evaluation-tools-list/submissions/ae311e10-03d4-11f0-b4ed-8357b43833d0.json

```
@@ -0,0 +1,64 @@
+{
+  "a11yloc": "https://www.barrierbreak.com/accessibility-statement/",
+  "actrules": "",
+  "assists": [
+    "Generating reports of evaluation results",
+    "Displaying information within web pages",
+    "Providing step-by-step evaluation guidance"
+  ],
+  "automated": [
+    "Component",
+    "Single page/screen",
+    "Restricted or password protected pages"
+  ],
+  "browsers": [
+    "Chrome"
+  ],
+  "comments": "",
+  "contact": "dev@barrierbreak.com",
+  "desktop": [
+    "Windows",
+    "MacOS"
+  ],
+  "features": "Detects failures, identifies issues to validate, and highlights code. Let A11yInspect assist your developers & accessibility professionals to evalaute for Web Content Accessibility Guideline (WCAG).\r\n\r\nOne-click highlights the issue along with the code. As simple as that!\r\nWide-range of Bookmarklets\r\nKey Code Attributes\r\nDownload Screenshots.",
```

The description contains typos, and would benefit from describing key features and functionalities without exclamation points.

For example:

```suggestion
  "features": "Detects failures, identifies issues to validate, and highlights code. A11yInspect assists your developers & accessibility professionals to evaluate for Web Content Accessibility Guidelines (WCAG).\r\n\r\nOne-click highlights the issue along with the code.\r\nWide-range of Bookmarklets\r\nKey Code Attributes\r\nDownload Screenshots.",
```